CONTRIBUTING.md

```diff
@@ -0,0 +1,53 @@
+# Contributing guidelines
+
+## Pull Request Checklist
+
+Before sending your pull requests, make sure you followed this list:
+
+- Read [Contributing Guidelines](CONTRIBUTING.md).
+- Read [Code of Conduct](CODE_OF_CONDUCT.md).
+- Refer to [Development process](#development-process).
+- Changes are consistent with the Coding Style of the project.
+- Build the project locally.
+- Run Unit Tests and ensure that you have the test coverage for your code.
+
+
+## Development process
+
+Please follow the steps below in order to make the changes:
+
+1. Clone the repository and open it in your favourite IDE.
+2. Make code changes to the core library of `graphql-java-codegen`.
+3. If changes are required in the plugin code, then **build** and **install** `graphql-java-codegen` first.
+
+   ```shell script
+   # This will install the library (including your recent changes) in your local maven repository.
+   ./gradlew clean install
+   ```
+   
+4. Build the plugin project with updated `graphql-java-codegen` library.
+
+   ```shell script
+   # Build Gradle plugin
+   cd plugins/graphql-java-codegen-gradle-plugin/graphql-codegen-gradle-plugin
```

Comment:
Is this correct or is there an extra `/graphql-codegen-gradle-plugin` here?

Reply:
That's correct. the module is named that way.